newIDE/app/src/Utils/Analytics/EventSender.js

```diff
@@ -44,7 +44,7 @@ const recordEvent = (name: string, metadata?: { [string]: any }) => {
   keenClient.recordEvent(name, metadata);
   posthog.capture(name, {
     ...metadata,
-    isInAppTutorialRunning: isUserflowRunning,
+    isInAppTutorialRunning: !!currentlyRunningInAppTutorial,
```

Comment:
can `currentlyRunningInAppTutorial` be null? should this be initiated to false instead?

Reply:
You're right, I changed it